app/components/Header.js

```diff
@@ -1,16 +1,20 @@
 
 import React from 'react';
 import { Link } from 'react-router';
+let bgImage = require('../assets/images/bg.png');
+let smallLogo = require('../assets/images/logo-small-white@3x.png');
```

Comment:
Might be cool to throw a js module somewhere, maybe under `/assets`, that provides an index of all images so we don't have this boilerplate.